ckanext/nhsengland/plugin.py

```diff
@@ -119,6 +119,7 @@ def before_map(self, route_map):
         """
         with routes.mapper.SubMapper(route_map,
             controller='ckanext.nhsengland.plugin:NHSEController') as m:
+            m.connect('dashboard', '/dash', action='dashboard')
```

Comment:
This got merged before anyone actually fixed it.

Recursive loop now on the production site.

Sad faces.